packages/next/build/webpack-config.ts

```diff
@@ -433,8 +433,7 @@ export default async function getBaseWebpackConfig(
             // are relative to requests we've already resolved here.
             // Absolute requires (require('/foo')) are extremely uncommon, but
             // also have no need for customization as they're already resolved.
-            const start = request.charAt(0)
-            if (start === '.' || start === '/') {
+            if (request.startsWith('.') || request.startsWith('/')) {
```

Comment:
```js
if (['.', '/'].some(char => request.startsWith(char))) {
  return callback()
}
```

Reply:
@Timer what about the above approach instead:question:

Reply:
May I suggest the following:
```javascript
if (/^[./]/.test(request)) {
```
The performance gain is probably negligible, but it's more concise.

Reply:
Yeah, much concise approach :+1: